src/move/tasks/analyze_latent.py

```diff
@@ -252,20 +397,80 @@ def analyze_latent(config: MOVEConfig) -> None:
         fig.savefig(fig_path, bbox_inches="tight")
         fig_df = pd.DataFrame(diffs, columns=cat_names[i], index=df_index)
         fig_df.to_csv(output_path / f"feat_importance_{dataset_name}.tsv", sep="\t")
+    logger.info(
+        "Analysis for categorical features completed. "
+        "Starting analysis for continuous features"
+    )
+
+    # NOW, THE SAME BUT FOR CONTINUOUS DATA
 
     for i, dataset_name in enumerate(config.data.continuous_names):
         logger.debug(f"Generating plot: feature importance '{dataset_name}'")
-        dataloaders = perturb_continuous_data(
-            test_dataloader, config.data.continuous_names, dataset_name, 0.0
-        )
-        num_features = len(dataloaders)
+        # NOT SURE IF IT WORKS THE SAME FOR CONTINUOUS FEATURES, CHECK THIS
+        # I did sth that did not work, I'll try again now.
```

Comment:
Not sure about leaving these comments. They would work better as GitHub issues.

I'm not even sure how to interpret them.

Reply:
I delete it

Reply:
deleted